README.md

```diff
@@ -22,110 +43,13 @@ ways to the Kubernetes cluster(s) that StarlingX manages:
 - as a StarlingX Application, which benefits from tight integration with the
   [StarlingX system](https://opendev.org/starlingx/config).
 
-## Pre-requisite
+## Tools requirements
 
 - Helm version 2+
 - Python version 3.8+
 - `pyyaml` version 6.0+
   - `$ pip3 install pyyaml==6.0.1`
 
-## General Overview
-
-![app flowchart](README/app-gen-tool.jpeg)
-
-## 3 Steps to create a Starlingx App
-
-### 1. Prepare Helm chart(s)
-
-#### What is Helm and a Helm chart?
-
-Helm is a Kubernetes package and operations manager. A Helm chart can contain
-any number of Kubernetes objects, all of which are deployed as part of the
-chart.
-
-The official place to find available Helm Charts is: https://artifacthub.io/.
-
-#### How to develop a Helm chart?
-
-Refer to official [helm doc](https://helm.sh/docs/)
-
-### 2. Create an app manifest
-
-A few essential fields are needed to create the app. The simplest
-example is:
-
-```yaml
-appManifestFile-config:
-  appName: stx-app
-  appVersion: 1.0.1
-  namespace: default
-  chart:
-    - name: chart1
-      version: 1.0.1
-      path: /path/to/chart1
-      chartGroup: chartgroup1
-  chartGroup:
-    - name: chartgroup1
-      chart_names:
-        - chart1
-
-setupFile-config:
-  metadata: 
-      author: John Doe
-      author-email: john.doe@email.com
-      url: johndoe.com
-      classifier: # required
-        - "Operating System :: OS Independent"
-        - "License :: OSI Approved :: MIT License"
-        - "Programming Language :: Python :: 3"
-```
```

Comment:
I think I'll miss an example, but since we're reducing even more this declarative file, I think I'm ok with removing it.